wurst/file/Encryptor.wurst

```diff
@@ -0,0 +1,13 @@
+package Encryptor
```

Comment:
Shouldn't this be more generic, like Processor/Pipeline/(De)serializer ?

Reply:
FWIW, I think this could be called (Persistable)Middleware.